lib/slack.ml

```diff
@@ -105,34 +98,28 @@ let generate_pull_request_notification ~slack_match_func ~(ctx : Context.t) ~get
            (string_of_pr_action action))
   in
   let thread = State.get_thread ctx.state ~repo_url:repository.url ~pr_url:html_url channel in
-  let%lwt summary =
-    Lwt.return
-    @@ sprintf "<%s|[%s]> Pull request #%d %s %s by *%s*" repository.url repository.full_name number
-         (pp_link ~url:html_url title) action sender.login
+  let summary =
+    sprintf "<%s|[%s]> Pull request #%d %s %s by *%s*" repository.url repository.full_name number
+      (pp_link ~url:html_url title) action_label sender.login
   in
+  let attachments = format_attachments ~slack_match_func ~footer:None ~body in
   let handler (res : Slack_t.post_message_res) =
     match notification.action with
-    | Opened | Ready_for_review | Labeled ->
+    | Opened | Ready_for_review | Labeled | Reopened ->
```

Comment:
I think new thread should be created on `Reopened` too, otherwise all subsequent activity after reopen will be orphaned.

Reply:
That makes sense, maybe it should be a comment ?